app/src/main/java/com/kylecombes/micrepeater/BluetoothStateReceiver.java

```diff
@@ -31,6 +28,9 @@ public void registerStateChangeReceiver(StateChangeReceiver receiver) {
 
     @Override
     public void onReceive(Context context, Intent intent) {
+        boolean mScoAudioConnected = false;
```

Comment:
I think this one should be outside of the method. If the battery status changes, for example, mScoAudioConnected will be false because the event is not related to the ScoAudo event, and so the status will be Disconnected because of other event changes.

Reply:
It wasn't being referenced anywhere else, so I just moved it inside this method. If you think it should be referenced somewhere else, can you please move it outside this method and add a reference? Thanks.

Reply:
It will be referenced again in this function. The point is when this function starts, `mScoAudioConnected` is defaulted to be false, which means the Status will be disconnected, let's say when the Bluetooth battery changes. We can make the variable static so it will be only inside this method, but that variable needs to persist.

Reply:
Oh I see what you're saying. My IDE misled me.